pkg/scheduler/reconciliation/reconciliation_test.go

```diff
@@ -729,3 +681,131 @@ func dbConnection(t *testing.T) db.Connection {
 	}
 	return dbConn
 }
+
+func TestReconciliationParallel(t *testing.T) {
+
+	t.Run("Create multiple instances of single reconciliations in parallel", func(t *testing.T) {
+		//initialize WaitGroup
+		var wg sync.WaitGroup
+		//reset db connection
+		dbConn = nil
+
+		repo := newPersistentRepository(t)
+		inventory, err := cluster.NewInventory(db.NewTestConnection(t), true, cluster.MetricsCollectorMock{})
+		require.NoError(t, err)
+
+		errChannel := make(chan error, 100)
```

Comment:
Make worth to use for  the lengh of this channel and for the loop-counter a variable to point out that both belong together

Reply:
👍🏻